sql/core/src/main/scala/org/apache/spark/sql/execution/debug/package.scala

```diff
@@ -50,23 +50,38 @@ package object debug {
     // scalastyle:on println
   }
 
+  /** Generate codegen debug info */
```

Comment:
Please add the same function description like what you did in `codegenStringSeq `